pom.xml

```diff
@@ -263,7 +264,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-enforcer-plugin</artifactId>
-                    <version>${version.plugin.enforcer}</version>
+                    <version>${version.plugin.maven-enforcer}</version>
```

Comment:
Seems the property name above in line 187 is missing "maven"

Reply:
That is the way it was already defined in the top-level pom under the banner "        <!-- maven plugin versions -->
"

Reply:
What I am saying is, you have renamed properties for "lib" to add maven at both places (property as well as usage) but for "plugin" you have just added it here in usage but definition of it is still without maven on line 187 i.e. ${version.plugin.enforcer}

Reply:
pushed